std/shared/src/main/scala/cats/effect/std/Random.scala

```diff
@@ -434,17 +434,20 @@ object Random extends RandomCompanionPlatform {
 
   }
 
-  private[std] abstract class ScalaRandom[F[_]: Sync](f: F[SRandom]) extends RandomCommon[F] {
+  private[std] abstract class ScalaRandom[F[_]: Sync](
+      f: F[SRandom],
+      hint: Sync.Type = Sync.Type.Delay)
+      extends RandomCommon[F] {
```

Comment:
Something like this should fix the CI failure.

```suggestion
  private[std] abstract class ScalaRandom[F[_]: Sync](
      f: F[SRandom],
      hint: Sync.Type)
      extends RandomCommon[F] {
      
      def this(f: F[SRandom]) = this(f, Sync.Type.Delay)
```

Reply:
Nice! Now CI is working.